tests/taskTests/codeDeployDeployApplication/codeDeployDeployApplication-test.ts

```diff
@@ -63,6 +71,14 @@ describe('CodeDeploy Deploy Application', () => {
     // TODO https://github.com/aws/aws-toolkit-azure-devops/issues/167
     beforeAll(() => {
         SdkUtils.readResourcesFromRelativePath('../../build/src/tasks/CodeDeployDeployApplication/task.json')
+
+        if (!fs.existsSync(tempDir)) {
+            fs.mkdirSync(tempDir)
+        } else {
+            fs.emptyDirSync(tempDir)
```

Comment:
Why not give every test its own unique folder to work in? This way you remove risk of affecting another concurrently running test.

Reply:
I guess if you set the TEMP env var with different values, that could affect other tests anyways...